DABI.js

```diff
@@ -0,0 +1,677 @@
+    var articles = new Array();
+    var items = new Object();
+    var nextLink;
+
+    //If Statement checks if page is a Search Result, then saves requested Items
+    if (detectWeb(doc, url) == "multiple") {
+        var links = doc.evaluate('//tr/td[1]/a', doc, nsResolver, XPathResult.ANY_TYPE, null);
+        var title = doc.evaluate('//tr/td[3]', doc, nsResolver, XPathResult.ANY_TYPE, null);
+
+        while (nextLink = links.iterateNext()) {
+            items[nextLink.href] = title.iterateNext().textContent;
+
+        }
+        items = Zotero.selectItems(items);
+        for (var i in items) {
+            articles.push(i);
+        }
+
+    } else if (detectWeb(doc, url) == "journalArticle") {
+        //saves single page items
+        articles = [url];
+    } else {
+        return false;
+    }
+
+    //Tells Zotero to process everything. Calls the "scrape" function to do the dirty work.
+    Zotero.Utilities.processDocuments(articles, scrape, function() {
+        Zotero.done();
+    });
+    Zotero.wait();
+    //Translator is FINISHED after running this line. Note: code doesn't run from top to bottom only.
+}
+
+//The function used to save well formatted data to Zotero
+function associateData(newItem, items, field, zoteroField) {
+    if (items[field]) {
+        newItem[zoteroField] = items[field];
+    }
+}
+
+function scrape(doc, url) {
+    //namespace code
+    var namespace = doc.documentElement.namespaceURI;
+    var nsResolver = namespace ? function(prefix) {
+        if (prefix == 'x') return namespace;
+        else return null;
+    } : null;
+
+    //variable declarations
+    var newItem = new Zotero.Item('journalArticle');
+    newItem.url = doc.location.href;
+    newItem.title = "No Title Found";
+
+    var items = new Object();
+    var headers;
+    var contents;
+    var link;
+    var blankCell = "temp";
+    var headersTemp;
+    var tagsContent = new Array();
+
+    var myXPathObject = doc.evaluate('//tr/th', doc, nsResolver, XPathResult.ANY_TYPE, null);
+    var myXPathObject2 = doc.evaluate('//tr/td', doc, nsResolver, XPathResult.ANY_TYPE, null);
+    var link = doc.evaluate('//tr/td/a', doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext();
+
+    //While Loop to populate "items" Object and save tags to an Array.
+    while (headers = myXPathObject.iterateNext()) {
+
+        headersTemp = headers.textContent;
+        if (!headersTemp.match(/\w/)) {
+            headersTemp = blankCell;
+            blankCell = blankCell + "1";
+        }
+
+        contents = myXPathObject2.iterateNext().textContent;
+
+        if (headersTemp.match("temp")) {
+            tagsContent.push(contents);
+        }
+
+        items[headersTemp.replace(/\s+/g, '')] = contents.replace(/^\s*|\s*$/g, '');
+
+    }
+
+    //Formatting and saving "title" fields
+    if (items["Titel"]) {
+        newItem.title = items["Titel"].replace(/\*/g, '');
+        var short = newItem.title.replace(/^(Die )|(Der )|(Das )/, '');
```

Comment:
Capturing parentheses are unnecessary. I think what you actually want is `/^(Die|(Der|Das) /`